lib/msf/ui/console/module_command_dispatcher.rb

```diff
@@ -105,7 +105,7 @@ def check_multiple(hosts)
         if exception.kind_of?(::Interrupt)
           raise exception
         else
-          elog("#{exception} #{exception.class}:\n#{exception.backtrace.join("\n")}")
+          elog('Error encountered with first Thread', error:e)
```

Comment:
I don't think `e` is defined here, it looks like it's meant to be `exception`

Reply:
Copypasta giveth, and copypasta taketh away